lib/zizia/record_importer.rb

```diff
@@ -24,6 +24,14 @@ def import(record:)
       raise e
     end
 
+    def import_all_records(records)
```

Comment:
Maybe add a comment that this method should be overwritten by a specific concrete importer